### PR TITLE
spelling error

### PR DESCRIPTION
**Pull request by hunylee**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - https://zeppelin.apache.org/contribution/contributions.html

### What type of PR is it?
[ Documentation ]

### Todos
* [x] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN/
* Put link here, and add [ZEPPELIN-*Jira number*] in PR title, eg. [ZEPPELIN-533]

### How should this be tested?
Outline the steps to test the PR here.

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

spelling error

responce => response

seprated => separated

### CODE REVIEW

File: docs/displaysystem/basicdisplaysystem.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ With `%html` directive, Zeppelin treats your output as HTML
 
 ## Table
 
-If you have data that row seprated by '\n' (newline) and column separated by '\t' (tab) with first row as header row, for example
+If you have data that row seperated by '\n' (newline) and column separated by '\t' (tab) with first row as header row, for example
```
Comment (khalidhuseynov) [inline docs/displaysystem/basicdisplaysystem.md]: I believe it should be `seperated`  -> `separated`

**Comment by khalidhuseynov** [comment]
Thanks for the contribution! could you modify PR description template with little info and address the comment.

**Comment by minahlee** [comment]
Thank you for the contribution. Having description would help reviewers to understand what you are trying to do. I believe you will do it from next time :)
Plus do you mind close #1547? I think it duplicates this one.
Meanwhile I will merge this in next 24 hours

File: docs/displaysystem/basicdisplaysystem.md (hunk 1 above)
Comment (hunylee) [inline docs/displaysystem/basicdisplaysystem.md]: thank you

**Pull request closed without merging**